tools/model-generation/spec/dataset.yml

```diff
@@ -43,18 +43,25 @@ fields:
     description: SciCat ID of the instrument used to measure the data.
     model_name: instrumentId
     type: str
+    used: [ raw ]
   - name: input_datasets
     description: Array of input dataset identifiers used in producing the derived dataset. Ideally these are the global identifier to existing datasets inside this or federated data catalogs.
     model_name: inputDatasets
     required: true
     type: List[PID]
     used: [ derived ]
   - name: investigator
-    description: Email of the (principal) investigator. The string may contain a list of emails, which should then be separated by semicolons.
-    model_name: { derived: investigator, raw: principalInvestigator }
+    description: Name(s) of the investigator(s). The string may contain a list of names, which should then be separated by semicolons.
+    model_name: investigator
     required: true
     type: str
-    validation: emails
```

Comment:
Does this PR contain everything from #63?

Reply:
No. If we merged #63, this should be resolved.